privaterelay/views.py

```diff
@@ -126,11 +126,12 @@ def waitlist(request):
         raise PermissionDenied
     json_body = json.loads(request.body)
     email = json_body['email']
-    if not email:
+    fxa_uid = json_body['fxa_uid']
+    if not email or not fxa_uid:
         return JsonResponse({}, status=400)
 
     _, created = Invitations.objects.get_or_create(
```

Comment:
Can't use `get_or_create` since this results into new `Invitations` record being created when the `fxa_uid` does not exist but have the same email address. We can `get_or_create` Invitations using `email` and `active=False`, check for `fxa_uid` value and then add the uid as needed?

Reply:
I'd like to merge this PR and https://github.com/mozilla/fx-private-relay/pull/345 before tackling this fix. I think this situation will come up if/when someone is on the Monitor wait-list AND directly joined the wait-list.

Reply:
Got it. Issues created here https://github.com/mozilla/fx-private-relay/issues/369